goal_src/engine/gfx/depth-cue.gc

```diff
@@ -5,3 +5,582 @@
 ;; name in dgo: depth-cue
 ;; dgos: GAME, ENGINE
 
+;; DECOMP BEGINS
+
+(define
+  *depth-cue-work*
+  (new 'static 'depth-cue-work
+    :texture-strip-tmpl
+    (new 'static 'dma-gif-packet
+      :dma-vif
+      (new 'static 'dma-packet
+        :dma
+        (new 'static 'dma-tag :qwc #x6 :id (dma-tag-id cnt))
+        :vif1
+        (new 'static 'vif-tag :imm #x6 :cmd (vif-cmd direct) :msk #x1)
+        )
+      :gif
+      (new 'static 'array uint64 2 #x50ab400000008001 #x43431)
```

Comment:
we could probably have nicer types for this now - this is probably a GIF tag.  At one point we simplified it because the decompiler/compiler had trouble with nested static bitfields/enums, but it should all work now.  Either way, not a big deal.